### ISSUE TITLE
https://nominatim-xbgmsharp.rhcloud.com/api/

### ISSUE THREAD

**Issue by penguin242** [xbgmsharp/piwigo-openstreetmap#164]
Is down?

No reverse lookup is possible

Error fetching geo reverse address data for 14746

**Comment by xbgmsharp** [comment]
That is correct, the service rhcloud.com went down. I was unable to find a free NodeJS/MongoDB app cloud provider. The app is available at https://github.com/xbgmsharp/nodejs-forecast.

**Comment by penguin242** [comment]
Why not using this link: https://nominatim.openstreetmap.org/reverse?format=json&lat=xxx&lon=xxxx&zoom=12

**Comment by Arnie97** [comment]
See also #149
